packages/xo-web/src/xo-app/menu/index.js

```diff
@@ -66,10 +71,20 @@ export default class Menu extends Component {
       window.removeEventListener('resize', updateCollapsed)
       this._removeListener = noop
     }
+
+    this._subscribeMissingPatches()
   }
 
   componentWillUnmount() {
     this._removeListener()
+    this._unsubscribeMissingPatches()
+  }
+
+  componentDidUpdate(prevProps) {
+    if (!isEqual(map(prevProps.hosts, 'id'), map(this.props.hosts, 'id'))) {
+      this.setState({ nMissingPatches: 0 })
```

Comment:
Why did you change this? Do you need the exact number of missing patches? Or only whether there are some or not?